wger/core/static/css/present_deactive_users.css

```diff
@@ -0,0 +1,39 @@
+/* Style the tab */
```

Comment:
This file name should be `present_deactivated_users` and not `present_deactive_users`. I think the word `deactive` doesn't exist yet :)

Reply:
@AGMETEOR I have renamed the file to `present_deactivated_users.css`